test/Microsoft.Extensions.FileProviders.Physical.Tests/Microsoft.Extensions.FileProviders.Physical.Tests.csproj

```diff
@@ -19,4 +19,14 @@
     <PackageReference Include="xunit.runner.visualstudio" Version="$(XunitVersion)" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Compile Update="MockFileSystemWatcher.cs">
+      <SubType>Component</SubType>
```

Comment:
(sigh). Hopefully this will go away one day. https://github.com/dotnet/roslyn-project-system/issues/182